src/flow_solver/flow_solver.h

```diff
@@ -3,16 +3,15 @@
 
 // for FlowSolver class:
 #include "physics/initial_conditions/initial_condition.h"
-#include "tests.h"
 #include "physics/physics.h"
 #include "parameters/all_parameters.h"
 
 // for generate_grid
+#include <deal.II/grid/tria.h>
 #include <deal.II/distributed/shared_tria.h>
 #include <deal.II/distributed/tria.h>
 
-// for the grid:
-#include "grid_refinement_study.h"
+// #include "grid_refinement_study.h" // remove?
```

Comment:
Commented line has not been removed. There's also a similar comment on line 36 of this file.

Reply:
fixed in most recent commit